types/storybook__react/index.d.ts

```diff
@@ -33,6 +33,7 @@ export function setAddon(addon: object): void;
 export function storiesOf(name: string, module: NodeModule): Story;
 export function storiesOf<T>(name: string, module: NodeModule): Story & T;
 export function forceReRender(): void;
+export function load(paths: __WebpackModuleApi.RequireContext, module: NodeModule): void;
```

Comment:
can we depend on the webpack-env types directly and import `RequireContext`? I just did a short search for `__WebpackModuleApi.RequireContext` in the DT repository and can't seem to find any other usage of this in that way.

Reply:
As far as I could tell, RequireContext isn't exported by webpack anywhere and I couldn't get a direct import working, but if you know of a different way I'm happy to switch it over!